cmd/create/environment.go

```diff
@@ -0,0 +1,62 @@
+package create
+
+import (
+	"os"
+	"strings"
+
+	"github.com/dream11/odin/internal/service"
+	environmentProto "github.com/dream11/odin/proto/gen/go/dream11/od/environment/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var envName string
+var provisioningType string
+var accounts string
+
+var environmentClient service.Environment
+
+// environmentCmd represents the environment command
+var environmentCmd = &cobra.Command{
+	Use:   "environment",
+	Short: "Create environment",
+	Args: func(cmd *cobra.Command, args []string) error {
+		return cobra.NoArgs(cmd, args)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		execute(cmd)
+	},
+}
+
+func init() {
+	environmentCmd.Flags().StringVar(&envName, "name", "", "name of the environment to be created")
+	environmentCmd.Flags().StringVar(&accounts, "account", "", "list of comma separated cloud provider accounts")
```

Comment:
Lets rename this to `accounts` since there can be multiple

Reply:
done